Vecherskiys/07/Task7.3.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html> 
+<HTML> 
+    <HEAD> 
+        <META charset="utf-8"> 
+        <TITLE>Ссылки. Оформление</TITLE> 
+		<style type="text/css">
+            table {border-collapse: collapse;}
+            td {
+                width: 50px;
+                height: 50px;
+                border: 1px solid black;
+            }
+            tr:nth-child(odd) td:nth-child(even){background-color: black}
```

Comment:
You can combine, remember about comma :)

Reply:
Ok. Complete:)